### PR TITLE
gingerblue: new port

### PR DESCRIPTION
**Pull request by oleaamot**
#### Description

GNOME Gingerblue is Free Software in development for musicians who want to compose, record and share original music to the Internet from the GNOME Desktop.

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 12.0.1 21A559 arm64
Xcode 13.1 13A1030d

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: gnome/gingerblue/Portfile [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                    port:zlib
+
+depends_run         port:adwaita-icon-theme \
+                    port:gstreamer1-gst-plugins-bad \
+                    port:gstreamer1-gst-plugins-good \
+                    port:gstreamer1-gst-plugins-ugly
+
+# reconfigure using autogen.sh from upstream git for intltool 0.51 compatibility
+
+configure.cmd       ./configure
```
Comment (reneeotten) [inline gnome/gingerblue/Portfile:59]: the comment and and the `configure.cmd` seem redundant, isn't `configure` the default command to do in the configure phase?

Comment (oleaamot) [reply gnome/gingerblue/Portfile]: I have removed the comment and the configure.cmd lines from the latest fork repository.

Comment (reneeotten) [reply gnome/gingerblue/Portfile]: thanks, can you please squash your commits?

Comment (herbygillot) [reply gnome/gingerblue/Portfile]: @oleaamot Unresolving this - can you please squash the 2 commits into a single one with the comment `gingerblue: new port`, as per @reneeotten's request?

Other than that I think this is good to go.

**Comment by oleaamot** [comment]
OK, it has been done.

See the latest fork on https://github.com/oleaamot/macports-ports/ <https://github.com/oleaamot/macports-ports/>

Best,
Ole

**Comment by reneeotten** [comment]
sorry, but this doesn't look correct as you still have your two commits and now also a merge commit. 

Please do a rebase with master to get rid of the latter one and then squash your two commits so that there is only one left.

**Comment by oleaamot** [comment]
How do I squash my two commits into one commit?  I tried git rebase -I HEAD~N

**Comment by pmetzger** [comment]
It is possible for us to do a "squash and merge" if necessary. (It may be that the submitter doesn't understand how to use git well enough to do what is being requested.)

**Comment by oleaamot** [comment]
Please "squash and merge" as I don't have the skills to do the squash of my two commits.

**Comment by pmetzger** [comment]
@oleaamot https://gist.github.com/longtimeago/f7055aa4c3bba8a62197

**Comment by reneeotten** [comment]
> How do I squash my two commits into one commit?  I tried git rebase -I HEAD~N

that is the right thing to use. You first need to do "git rebase upstream/master" to get rid of the merge commit. Then you'd do "git rebase -i HEAD~2" which will show your last two commits. Change "pick" of the last one to "fixup",  save-and-close the editor and you should have only one commit left. Since you now effectively rewrote your git history you'll need to force-push to your fork ;"git push origin -f", assuming that 'origin' is your fork).

**Comment by reneeotten** [comment]
> It is possible for us to do a "squash and merge" if necessary. (It may be that the submitter doesn't understand how to use git well enough to do what is being requested.)

sure, but it's a useful skill to have and not *that* complicated

**Comment by mascguy** [comment]
Ole, if you ever get frustrated with Git, we can help you cleanup the commits. There's no need to close a PR, and open a new one, though.

**Pull request closed without merging**